docs/topics/native/native-cocoapods-libraries.md

```diff
@@ -285,4 +285,29 @@ kotlin {
         }
     }
 }
+```
+
+### Dependencies between Pods
+
+You can use the cinterop binding generated for another Pod when building a binding for the new Pod.
+The dependent Pod should be declared before setting up the dependency. The Kotlin CocoaPods Gradle plugin supports two ways
+of adding dependencies between Pods:
+
+* `useInteropBindingFrom()` function specifies the name of the existing Pod that is used as dependency.
+* `interopBindingDependencies` contains a list of all dependencies to other Pods.
+
+```kotlin
+pod("WebImage") {
+    version = "~> 1.0.0"
```

Comment:
I'd suggest to stop promoting `~>` versions. We support them, but the Gradle users are far more used to the fixed versions.